force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,18 +105,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
+        Integer result = 0;
+        for (Integer i = 0; i < str.length(); i++)  {
             // get individual characters from the string - hint use substring or split
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
+            String c = str.substring(i, i + 1);
+            
+            if (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u')  {
```

Comment:
If there are this many to check and it is all OR conditions, it is probably better to use a Set or List